src/components/ContributorTasks/ContributorTasks.test.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
can you pull in the latest changes from `development`, and then remove this `eslint-disable`?